core/repository/transaction/pg.ts

```diff
@@ -56,8 +56,9 @@ class TransactionPGRepo implements ITransactionPGRepository<IAsset> {
         });
     }
 
-    async deleteById(trsId: string | Array<string>): Promise<void> {
-        await db.none(queries.deleteByIds, [trsId]);
+    async deleteById(trsId: string | Array<string>): Promise<Array<string>> {
+        const response = await db.manyOrNone(queries.deleteByIds, [trsId]);
```

Comment:
Will it works correctly when trsId will be string and array of string?

Reply:
Tests cover both variants